tests/unit/progress.py

```diff
@@ -6,15 +6,15 @@
 
 class TestProgressAware(TestCase):
     @mock.patch("sys.stdout.isatty", return_value=True)
-    @mock.patch("dvc.progress.Progress._print")
+    @mock.patch("dvc.progress.Progress.print")
     def test(self, mock_print, mock_isatty):
         function = progress_aware(lambda: None)
 
         function()
-        mock_print.assert_not_called()
+        mock_print.assert_called_once()
 
         progress.update_target("testing", 0, 100)
         function()
-        mock_print.assert_called()
+        self.assertEqual(4, mock_print.call_count)
```

Comment:
Why should it be called 3 more times though? One for clearln, then print itself and then what? Could you add a comment about it please?

Reply:
Ill add comment